yowza/yowza/SDKDetailsViewController.swift

```diff
@@ -62,6 +60,7 @@ class SDKDetailsViewController: UIViewController {
         BITHockeyManager.shared().authenticator.webpageURL = URL(string: config.environment.rawValue)
         BITHockeyManager.shared().authenticator.restrictApplicationUsage = config.restrictApplicationUsage
         BITHockeyManager.shared().authenticator.authenticateInstallation()
+        BITHockeyManager.shared().authenticator.restrictionEnforcementFrequency = .onAppActive
```

Comment:
I'm assuming this setting is only applied if `restrictApplicationUsage` is `true`